pry-remote-em.gemspec

```diff
@@ -8,12 +8,13 @@ Gem::Specification.new do |s|
   s.homepage      = 'http://github.com/simulacre/pry-remote-em'
```

Comment:
this should now be `'https://github.com/gruis/pry-remote-em'`

Reply:
Done.